src/Integration/MefServices/IHost.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace SonarLint.VisualStudio.Integration
 {
-    internal interface IHost : IServiceProvider
```

Comment:
This is the root of all of the other changes: `IHost` no longer acts as a pass-through for `IServiceProvider`, so some components will now need to be given an `IServiceProvider` as well as an `IHost`.